src/w3d/lib/random.cpp

```diff
@@ -171,7 +171,7 @@ int Random4Class::operator()(int minval, int maxval)
 float Random4Class::Get_Float()
 {
     int x = (*this)();
-    unsigned int *y = (unsigned int *)&x;
+    unsigned int *y = reinterpret_cast<unsigned int *>(&x);
```

Comment:
again this is just plain weird. There is no need to use a pointer here `auto y = static_cast<int32_t>(x);`

Reply:
Same as above.